### ISSUE TITLE
Incorrect data from analog sensors - reset system

### ISSUE THREAD

**Issue by JoshSK** [burgrp/nanook-mb#2]
Hello,

sometimes I get incorrect data from analog sensor boards, cold side but also hot side, the only one option is to turn off the main switch.
This is very annoying when I'm not at home. I'm analyzing if it could be caused by EMI or power supply.

Could analog boards have a implemented restart command  via I2C or is there no way to restart the system without a hard reset?

Thanks

![Snímka obrazovky zo dňa 2024-11-21 21-39-05](https://github.com/user-attachments/assets/d9d3dc41-f1b3-443d-b01f-0ba07665c66c)

**Comment by burgrp** [comment]
Hi, although rarely, I'm facing this issue too. I suspect it is a firmware bug, so it could be fixed. The restart command you suggest could be a solution, but we should try to find the root cause.

**Comment by JoshSK** [comment]
Of course, I agree. I see reset as a short-term solution. I personally suspect a problem with vref. It might be worth considering testing the firmware with a fixed voltage. But maybe I'm thinking wrong...
